src/main/java/com/artipie/http/MainSlice.java

```diff
@@ -123,19 +123,16 @@ private static Optional<YamlMapping> metricsStorage(final Settings settings) {
 
     /**
      * Checks that metrics are collected by Prometheus.
+     *
      * @param settings Artipie settings
-     * @return Yaml node, could be null
+     * @return True if metrics are collected by Prometheus.
      */
     private static boolean isPrometheusConfigAvailable(final Settings settings) {
-        final Optional<YamlMapping> myml =
-            Optional.ofNullable(settings.meta().yamlMapping("metrics"));
-        final boolean available;
-        if (myml.isPresent()) {
-            final String type = myml.get().string("type");
-            available = MetricsFromConfig.PROMETHEUS.equals(type);
-        } else {
-            available = false;
+        boolean res = false;
+        final YamlMapping metrics = settings.meta().yamlMapping("metrics");
+        if (metrics != null) {
+            res = MetricsFromConfig.PROMETHEUS.equals(metrics.string("type"));
```

Comment:
hm, this is actually my mistake, I forgot to fix this part when make `metrics` section yaml sequence instead of yaml mapping. Please, check metrics settings descriptions in [wiki](https://github.com/artipie/artipie/wiki/Configuration-Metrics) and let's correct this method accordingly...